packages/web3-eth/src/utils/decode_signed_transaction.ts

```diff
@@ -14,12 +14,14 @@ GNU Lesser General Public License for more details.
 You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
-import { TransactionFactory } from '@ethereumjs/tx';
+import defaultImport, * as fullImport from '@ethereumjs/tx';
```

Comment:
Doing this doesn't increase the build size?

Reply:
@jdevcs But would this affect tree shaking since we're now importing everything and setting it to a `const` instead of just importing and using the `TransactionFactory` export?

Reply:
I think this may not affect treeshaking as in case of ESM usage of our lib , dependency usage tree is still construable for module being used. 
```
import defaultImport, * as fullImport from '@ethereumjs/tx';
const { TransactionFactory } = defaultImport || fullImport;
```
Even if any issue arise , the lib under discussion is decided to be removed, or we can upgrade this to latest @ethereumjs/tx where these exports are fixed as mentioned earlier:

> This export problem is fixed in @ethereumjs/tx in 4.x and we can upgrade to that in our 4.x after reviewing breaking changes in another PR.